src/ts/content.ts

```diff
@@ -116,24 +124,32 @@ const replaceIcon = ({
     isDirectory = iconDom.classList.contains('octicon-file-directory');
   }
 
-  const className: string | null = colorsDisabled
-    ? fileIcons.getClass(filename)
-    : fileIcons.getClassWithColor(filename);
+  const getClassName = () => {
+    if (colorsDisabled) {
+      return getClass(filename);
+    }
+
+    if (darkMode) {
+      return getClassWithDarkColor(filename);
```

Comment:
I'm not sure about this. Usually icon color is brighter in dark mode. Actually I'm okay with following their color themes, but the original "dark mode" of this extension may need to be changed to like "bright color mode". Would you like to implement it as well? Or I can do it after this PR is merged.

Reply:
I'm not exactly sure what you mean, are you talking about adding another option for brightening the icons?  You may want to do it to avoid confusion on this part, because I'm not exactly sure how you'd like this implemented.

Reply:
One potential solution is to determine whether the user has a light or dark theme going for GitHub and use that for the icon packs built in light/dark mode.  And then put your original light/dark mode toggle back in the way it was before.  The package [`github-material-icons-extension`](https://github.com/Claudiohbsantos/github-material-icons-extension) does something like that and it would be pretty straightforward to implement if you'd like me to.

Reply:
Ok I did what I mentioned, using the site's dark mode setting for the icon packs dark mode and returned the dark mode toggle to the way it was working before.  Let me know if this works!